spec/components/notification_message_component_spec.rb

```diff
@@ -1,111 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe NotificationMessageComponent do
```

Comment:
Do any of these tests now belong in `FlashMessageComponent`? (There is some extra logic in `FlashMessageComponent` that we're not testing other than in system specs).

Reply:
The only extra tests needed related to adding a link. But I’ve removed this functionality in 91b8d5be7c857a60ac240d4c8dcab98d2506ee96 because the one place we were using it, turns out we no longer need it.